.github/workflows/build.yml

```diff
@@ -35,7 +37,7 @@ jobs:
         id: get-product-version
         run: |
           echo "::set-output name=product-version::$(make version)"
-          echo "::set-output name=ldflags::-s -w -X github.com/hashicorp/terraform-aws-consul-lambda/version.GitCommit=$(git rev-parse --short HEAD)"
```

Comment:
I took this out because it is not being used. I think this was probably copied from the `consul-ecs` repo where the version of the GitCommit was getting injected into the CLI via this flag. We don't have a CLI for the registrator or the extension so I removed it.